.codecov.yml

```diff
@@ -1,11 +1,19 @@
 codecov:
+  branch: dev
+
+coverage:
+  # https://docs.codecov.com/docs/coverage-configuration
+  range: "80...90"
   # https://docs.codecov.io/docs/commit-status
   status:
+    # TODO: re-enable patch in the future
+    patch: off
```

Comment:
What's this used for?

Reply:
When enabling `github_checks.annotations`, there are 2 checks by default:

- `codecov/project`
- `codecov/patch`

I'm not totally sure about the differences, but for the total coverage check is made on `codecov/project`, whereas on `codecov/patch` it checks, for instance, that the diff of the branch reaches the target coverage (for instance, if the PR only adds 1 line, that does not change the total coverage by much `codecov/project` will pass but `codecov/patch` will fail because it will have a "patch coverage" of 0%).

I'm proposing that we start with `codecov/patch` disabled, _should_ have the exact same behavior as we do by checking the coverage when running tests (it doesn't check the patch coverage). And only after we're comfortable with dealing with codecov we would enable `codecov/patch`, which should be much more useful in practice than only looking at the total coverage.